client/src/components/food/multi-select-bar.test.tsx

```
@@ -0,0 +1,251 @@
+
+vi.mock('@tanstack/react-query', async (importOriginal) => {
+  const actual = await importOriginal();
+  return {
+    ...actual,
+    // Default mock for useMutation, can be overridden per test if needed for specific isPending states
+    useMutation: vi.fn((options) => {
+      // Determine which mutate function to return based on context (e.g., mutationFn)
+      // This is a simplified approach. For real tests, you might spy on options.mutationFn.
+      // For this component, we have 3 mutations. We'll need to differentiate them.
+      // A common way is to mockImplementation for useMutation in beforeEach or per test.
+      if (options && options.mutationFn) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)